blog/23-12-22- how-to-become-a-web3-developer/how-to-be-a-web3-developer.md

```diff
@@ -0,0 +1,193 @@
+---
+title: How to become a Web3 Developer
+description: In this tutorial, we will show you how to become a web3 developer and build decentralized applications on the Celo blockchain.
+authors:
+  - name: Elijah Sorinola
+    title: Content Writer, Celo Sage
+    url: https://twitter.com/ElijahSorinola
+    image_url: blog/23-12-22- how-to-become-a-web3-developer/images/web3 32.png
```

Comment:
I noticed there is a thumbnail missing

Reply:
hey @izzecode can you add a thumbnail(png file in the Trello card) in this directory; src/data-tutorials/showcase/beginner/

otherwise, great work